.travis.yml

```diff
@@ -50,7 +50,7 @@ script:
     - if [[ $TRAVIS_OS_NAME == "linux" ]] && [[ $PYTHONVER == 3.6 ]]; then
           coverage run --source=exatomic setup.py test;
           rm -rf docs/source/*.txt;
-          sphinx-apidoc -eM -s txt -o docs/source/ exatomic/;
+          sphinx-apidoc -eM -s txt -o docs/source/ exatomic/ */tests/* ;

```

Comment:
This is really a good idea. Can we be even more greedy and do `*test*`?

Reply:
Makes sense